libs/order/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@daffodil/order",
+  "version": "0.0.0-PLACEHOLDER",
+  "description": "A package for interacting with platforms that provide geographic data information like States and Countries",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/graycoreio/daffodil"
+  },
+  "author": "Graycore LLC",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/graycoreio/daffodil/issues"
+  },
+  "scripts": {
+    "build": "ng build order --prod",
+    "lint": "ng lint order",
+    "test": "ng test order --watch=false --browsers=ChromeHeadless",
+    "publish": "(cd ../../dist/order; npm publish --access=public)"
+  },
+  "homepage": "https://github.com/graycoreio/daffodil",
+  "peerDependencies": {
+    "@angular/core": "0.0.0-PLACEHOLDER",
+    "@daffodil/core": "0.0.0-PLACEHOLDER"
+  },
+  "devDependencies": {
+    "@daffodil/core": "0.0.0-PLACEHOLDER",
+    "@daffodil/cart": "0.0.0-PLACEHOLDER",
+    "@daffodil/geography": "0.0.0-PLACEHOLDER",
+    "@daffodil/driver": "0.0.0-PLACEHOLDER"
+  },
+  "optionalDependencies": {
+    "@daffodil/driver": "0.0.0-PLACEHOLDER"
```

Comment:
I don't think this dependency is needed. Is it?

Reply:
no, I don't think it is. changed